lib/asciidoctor/opal_ext/file.rb

```diff
@@ -135,6 +135,46 @@ def self.read(path)
         }
         return data;
       )
+    # afx means asciidocfx
+    when 'javafx'
+      %x(
+        var data = '';
+
+        if(afx) {
```

Comment:
The same as [here](https://github.com/asciidoctor/asciidoctor/pull/1353/files#r30944891), `if` should be followed with one space.